components/collector/tests/source_collectors/quality_time/test_missing_metrics.py

```diff
@@ -165,3 +167,18 @@ async def test_subject_without_overridden_subject_name(self):
             total=str(self.nr_supported_metric_types()),
             value=str(len(entities)),
         )
+
+    async def test_source_types_to_include(self):
+        """Test that the number of non-ignored missing metrics is returned when filtered by source type."""
+        self.set_source_parameter("source_types_to_include", ["Jira"])
```

Comment:
Should we not also have test cases which deal with a combination of include and ignore, or does that functionally not make sense?

Reply:
The combination would be the intersection of both filters because they are applied sequentially. But it can't harm to add a test.